extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -128,17 +129,24 @@ describe('amp-a4a', () => {
     resetScheduledElementForTesting(window, 'amp-a4a');
   });
 
-  function createA4aElement(doc) {
+  function createA4aElement(doc, opt_height) {
+    if (opt_height == undefined) {
+      opt_height = 50;
+    }
     const element = createElementWithAttributes(doc, 'amp-a4a', {
       'width': '200',
-      'height': '50',
+      'height': String(opt_height),
       'type': 'adsense',
     });
     element.getAmpDoc = () => {
       const ampdocService = ampdocServiceFor(doc.defaultView);
       return ampdocService.getAmpDoc(element);
     };
     element.isBuilt = () => {return true;};
+    element.getLayoutBox = () => {
+      const visible = element.style.display != 'none';
+      return layoutRectLtwh(0, 0, visible ? 200 : 0, visible ? opt_height : 0);
```

Comment:
Unfortunately the way we're initializing this test, it will fail without explicitly adding getLayoutBox (presumably this is why other getters have been attached to element).  Perhaps we need to rework the tests but I would rather not delay this PR.

In any event, I reworked so that rect is passed as optional item but in the end display none was not actually being tested.  Instead I reworked to verify behavior with 0 height or width.